testdata/scripts/scope.txt

```diff
@@ -0,0 +1,46 @@
+go build main.go
+binsubstr main$exe 'globalVar' # 'globalType' only matches on go < 1.15 
+! binsubstr main$exe 'localConst' 'localShort' 'localVar' 'localType' 'funcParam' 'namedReturn' 'globalConst'
+exec ./main
+cmp stderr main.stderr
+
+garble -debugdir=debug build main.go
+! binsubstr main$exe 'localConst' 'localShort' 'localVar' 'localType' 'funcParam' 'namedReturn' 'globalConst' 'globalVar' 'globalType'
```

Comment:
if you want to make your life easier, you could name those `localNameConst` and `localNameVar` and so on, then the regexp can just check that `localName` is missing.

Reply:
I like it this way so if something breaks we know exactly which identifier is to blame